assets/translations/en-us.yaml

```diff
@@ -4707,7 +4707,7 @@ typeDescription:
   monitoring.coreos.com.prometheus: A Prometheus server is a Prometheus deployment whose scrape configuration and rules are determined by selected ServiceMonitors, PodMonitors, and PrometheusRules and whose alerts will be sent to all selected Alertmanagers with the custom resource's configuration.
   monitoring.coreos.com.alertmanager: An alert manager is deployment whose configuration will be specified by a secret in the same namespace, which determines which alerts should go to which receiver.
   node: The base Kubernetes Node resource represents a virtual or physical machine which hosts deployments. To manage the machine lifecyle, if available, go to Cluster Management.   
-  catalog.cattle.io.clusterrepo: 'A chart repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo: 'A cluster repository is a Helm repository, {vendor} git based application catalog or a rke template repository. It provides the list of available charts and templates in the clusters.'
```

Comment:
Can we have two messages - only repositories in the local cluster can contain cluster templates.

For the local cluster, user the new message, for non-local clusters, use the old message.